services/src/main/java/org/exoplatform/news/utils/NewsUtils.java

```diff
@@ -123,6 +124,15 @@ public static boolean canPublishNews(org.exoplatform.services.security.Identity
   public static boolean canDeleteTargetNews(org.exoplatform.services.security.Identity currentIdentity) {
```

Comment:
```suggestion
  public static boolean canManageNewsPublishTargets(org.exoplatform.services.security.Identity currentIdentity) {
```
to be used for all manage operations authorizations